app/presenters/step_by_step_page_presenter.rb

```diff
@@ -9,7 +9,7 @@ def initialize(step_by_step_page)
 
   def summary_metadata
     items = {
-      "Status" => step_by_step_page.status[:text],
+      "Status" => step_by_step_page.status.humanize,
```

Comment:
Is the capitalisation of `unpublished_changes` to "Unpublished changes" correct? or should it be "Unpublished Changes"

Reply:
The [original text](https://github.com/alphagov/collections-publisher/blob/master/app/views/step_by_step_pages/index.html.erb#L83) is "Unpublished changes", so I assumed that's it's alright.

Reply:
👍